rosidl_gen/idl_generator.js

```diff
@@ -258,15 +296,19 @@ async function generateActionJSStruct(actionInfo, dir) {
 }
 
 async function generateJSStructFromIDL(pkg, dir) {
-  await Promise.all([
-    ...pkg.messages.map((messageInfo) =>
-      generateMessageJSStruct(messageInfo, dir)
-    ),
-    ...pkg.services.map((serviceInfo) =>
-      generateServiceJSStruct(serviceInfo, dir)
-    ),
-    ...pkg.actions.map((actionInfo) => generateActionJSStruct(actionInfo, dir)),
-  ]);
```

Comment:
I found this regression introduced by https://github.com/RobotWebTools/rclnodejs/pull/726, it caused the test fails because the JS message files haven't been generated yet when the `npm test` starts. I think we need to roll out this fix asap.

We need to put all the `Promise`s into an arry here and pass it to `Promise.all()`